pkgs/top-level/all-packages.nix

```diff
@@ -34728,4 +34728,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  qtrvsim = libsForQt5.callPackage ../applications/science/computer-architecture/qtrvsim {};
```

Comment:
`all-packages.nix` should be mostly sorted - see the comment at the top.

Reply:
~fixed (the q letter area is not well sorted)